extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -878,6 +898,11 @@ export class AmpA4A extends AMP.BaseElement {
         this.protectedEmitLifecycleEvent_('iframeAlreadyExists');
         return Promise.resolve();
       }
+      if (this.isFluid) {
+        return this.creativeBody_
+            ? this.renderViaNameAttrOfXOriginIframe_(this.creativeBody_)
```

Comment:
Break after the operator.

Reply:
Seems uglier that way to me. Is this a strictly enforced convention? Because the linter doesn't seem to mind.